src/subcommands/projects.ts

```diff
@@ -0,0 +1,272 @@
+import { Args } from "../args.ts";
+import { API, APIError, endpoint } from "../utils/api.ts";
+import TokenProvisioner from "../utils/access_token.ts";
+import { wait } from "../utils/spinner.ts";
+import { Organization, Project } from "../utils/api_types.ts";
+import { bold, green, magenta, red } from "../../deps.ts";
+import { error } from "../error.ts";
+import organization from "../utils/organization.ts";
+
+const help = `Manage projects in Deno Deploy
+
+USAGE:
+    deployctl projects <SUBCOMMAND> [OPTIONS]
+
+SUBCOMMANDS:
+    list            List the name of all the projects accessible by the user
+    show            View details of a project. Specify the project using the --project option; otherwise, it will 
+                    show the details of the project specified in the config file or try to guess it from the working context
+    delete          Delete a project. Specify the project in the same way as the show subcommand
+    create          Create a new project. Specify the project name in the same way as the show subcommand 
+    rename <NAME>   Change the name of the project. Specify the project in the same way as the show subcommand
+
+
+OPTIONS:
+    -h, --help                    Prints this help information
+    -p, --project=<NAME|ID>       The project selected. 
+        --org=<ORG>               Specify an organization. When creating a project, defaults to the user's personal organization.
+                                  When listing projects, use "personal" to filter by the personal organization.
+        --token=<TOKEN>           The API token to use (defaults to DENO_DEPLOY_TOKEN env var)
+        --config=<PATH>           Path to the file from where to load DeployCTL config. Defaults to 'deno.json'
+        --color=<auto|always|off> Enable colored output. Defaults to 'auto' (colored when stdout is a tty)
```

Comment:
```suggestion
```
I wonder if we need a flag for it since we have a standard way to disable color by using `NO_COLOR=true`?

Reply:
Yeah I saw Deno already supports the `NO_COLOR` env. However, I have 2 reasons to justify the addition of the flag:
1. When stdout is not a tty, it's _generally_ preferred not to have color. However, there are exceptions to this (for example, the command is part of a pipe that ends in a tty). The ideal UX (IMO) is one that defaults to the most common case but gives the user the option to change the behavior. You cannot do that with `NO_COLOR` env.
2. In my personal experience, using `--color=<auto|always|never>` is a fairly common* way to control coloring in CLI commands, and it's what I intuitively expect/would like to find. I hadn't heard about the `NO_COLOR` env at all. I'd like to think I'm not the only one with the same experience 😛 

* `ls`, `grep`, `git`, `diff`, `cargo`...